test/cri-containerd/container_virtual_device_test.go

```diff
@@ -206,10 +206,10 @@ func Test_RunContainer_VirtualDevice_GPU_LCOW(t *testing.T) {
 
 	testDeviceInstanceID, err := findTestNvidiaGPUDevice()
 	if err != nil {
-		t.Fatalf("skipping test, failed to find assignable nvidia gpu on host with: %v", err)
+		t.Skipf("skipping test, failed to find assignable nvidia gpu on host with: %v", err)
```

Comment:
I don't think this was intentional was it?

Reply:
It was, shouldn't we be doing what it says in the description and skipping instead of failing the test?

Reply:
No, these were changed specifically to `Fatal` since we now use features to determine what tests to run, so these would only be run intentionally. In which case, they should fail if the machine doesn't have the correct resources. 

It sounds like instead the comments should be updated to reflect that. But either way, that should be done in a separate PR.

Reply:
Ok gotcha the comment was tripping me up. I'll revert these!

Reply:
I ended up fixing some of the ones that actually were Skip, and just changed ones that didn't have any formatting needed to just Fatal from Fatalf